contrib/babelfishpg_tsql/src/pltsql_ruleutils.c

```diff
@@ -2836,3 +2837,20 @@ tsql_printTypmod(const char *typname, int32 typmod, Oid typmodout)
 	return res;
 }
 
+char*
+return_collation_name(Oid collOid)
```

Comment:
generate_tsql_collation_name would be better? and also, add a comment to explain the function.

Reply:
Done